templates/custom/media/media--image--gallery-multiple.html.twig

```diff
@@ -28,8 +28,7 @@
     <div class="image-wrapper" data-ratio="8:5">
       {{ content.field_media_image }}
     </div>
-    {% if content.field_media_caption|render|trim is not empty %}
-      <figcaption>{{ content.field_media_caption }}</figcaption>
-    {% endif %}
+
+    {{ content.field_media_caption }}
```

Comment:
No check necessary?

Reply:
Drupal doesn't render anything when it's empty, right? The extra checks are only necessary when extra markup is added in the template.